profiles/bicycle.lua

```diff
@@ -211,19 +211,20 @@ function way_function (way, result)
   local bicycle = way:get_value_by_key("bicycle")
 
   -- name
-  if ref and "" ~= ref and name and "" ~= name then
-    result.name = name .. " (" .. ref .. ")"
-  elseif ref and "" ~= ref then
-    result.name = ref
-  elseif name and "" ~= name then
+  if name and "" ~= name then
     result.name = name
   -- TODO find a better solution for encoding way type
```

Comment:
what about this todo? Way types are encoded separately now. Anything else missing here?